cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -197,7 +197,10 @@ remove_docker(){
 			sudo dnf -y remove ${pkg_name}
 		elif [ "$ID" == "centos" ] || [ "$ID" == "rhel" ]; then
 			sudo yum -y remove ${pkg_name}
-		elif [[ "$ID" =~ ^opensuse.*$ ]] || [ "$ID" == "sles" ]; then
+		elif [[ "$ID" =~ ^opensuse.*$ ]]; then
+			sudo zypper -n remove ${pkg_name}
+		elif [ "$ID" == "sles" ]; then
+			sudo zypper removelock docker
```

Comment:
`s/docker/${pkg_name}/` ?

Reply:
@devimc you need to put `docker` because `pkg_name` could include different packages like  `docker-libnetwork-0.7.0.1+gitr2664_3ac297bc7fd0-13.1.x86_64, docker-runc-1.0.0rc5+gitr3562_69663f0bd4b6-1.9.1.x86_64, etc`